selenium/src/test/java/au/org/emii/portal/tests/ImosLinkCheckTest.java

```diff
@@ -0,0 +1,38 @@
+package au.org.emii.portal.tests;
+
+import au.org.emii.portal.utils.SeleniumUtil;
+import au.org.emii.portal.utils.WebElementUtil;
+import org.apache.log4j.Logger;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+//Link to IMOS website
+public class ImosLinkCheckTest extends BaseTest {
+
+    private static Logger log = Logger.getLogger(ImosLinkCheckTest.class.getName());
+
+    //Click on the IMOS link at the top right of the screen (22/08 now at bottom of screen)
+    @Test(invocationCount = invocationCount, threadPoolSize = threadPoolSize)
+    public void ImosLinkCheckTest() {
+        // Go to home page
+        getDriver().get(AODN_PORTAL_HOME_PAGE);
+        int invalidLinksCountHomePage = SeleniumUtil.validateInvalidLinks(getDriver());
```

Comment:
Seem to be passing the driver to every invocation of methods on this class - may be better to instantiate the class with the driver up front and change static methods to instance methods so you don't have to pass it all the time.